app/components/mandatees/checkbox-select.hbs

```diff
@@ -2,33 +2,15 @@
   <Auk::Loader />
 {{else if this.ministers}}
 <div class="auk-form-group">
-  <div class="au-u-flex au-u-flex--between au-u-flex--vertical-center au-u-margin-bottom-small">
-    <Auk::Label for="ministerName">
-      {{t "ministers"}}
-    </Auk::Label>
-    <p>
-      <AuButton
-        {{on "click" this.toggleAll}}
-        @skin="link"
-      >
-        {{#if this.allSelected}}
-          {{capitalize (t "deselect-all")}}
-        {{else}}
-          {{capitalize (t "select-all")}}
-        {{/if}}
-      </AuButton>
-    </p>
-  </div>
-  <Auk::CheckboxList class="au-c-checkboxlist-column">
-    {{#each this.ministers as |minister|}}
-      <AuControlCheckbox
-        @checked={{this.isChecked minister}}
-        @label={{minister.fullName}}
-        @onChange={{fn this.toggle minister}}
-      />
-    {{/each}}
-  </Auk::CheckboxList>
-  </div>
+  <Auk::CheckboxTree
+    @label={{t "ministers"}}
+    @layout="grid"
+    @items={{this.ministers}}
+    @itemStructure={{hash label="fullName"}}
+    @selectedItems={{this.selectedMinisters}}
+    @didUpdate={{this.onChangeMinisters}}
+  />
```

Comment:
Mini detail en helemaal niet zo belangrijk, maar nu staan de select boxes naar rechts geïndenteerd t.o.v. "Ministers". Kan die indentatie (gemakkelijk) hier ongedaan gemaakt worden? Het kan zeker ook zo door, 't is eerder om te weten of het mogelijk is :smile: 

![image](https://user-images.githubusercontent.com/22411874/233598993-cc3e9fa8-de63-4173-8858-3518f9f9bda8.png)
vs.
![image](https://user-images.githubusercontent.com/22411874/233599056-f35650e9-885b-4c77-af9a-96e7c93ce8f3.png)

Reply:
Die indentatie zit eigenlijk in het component gebakken, maar is in theorie wel overschrijfbaar. Nu, lijkt me momenteel misschien wel niet de effort waard? 😅

Ik kan het desnoods wel eens voorleggen bij Johan hoe hij dat ziet.

Reply:
Als het ingebakken is, dan mag het zeker zo blijven! Inderdaad niet de moeite waard :)